2019-08-14 08:46:18: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 08:47:47: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 09:15:35: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 09:16:05: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 09:43:31: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 09:51:46: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 09:55:27: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:04:06: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:07:42: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:10:10: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:11:39: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:12:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:12:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:12:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:12:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:12:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:12:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:12:51: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 10:27:16: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:06:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:44:26: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:44:26: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:44:26: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 11:44:26: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 12:25:06: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 12:25:06: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 12:25:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 12:25:20: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 13:58:40: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:34:41: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:35:18: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:36:32: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:36:43: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:36:43: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:36:43: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:36:43: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:36:43: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:36:43: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:47:19: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 14:54:55: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 15:13:00: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 15:20:01: 由于连接方在一段时间后没有正确答复或连接的主机没有反应，连接尝试失败。   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:44: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
2019-08-14 16:06:45: Cannot access a disposed object.
Object name: 'System.Net.Sockets.Socket'.   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult, SocketError& errorCode)
   at System.Net.Sockets.Socket.EndReceive(IAsyncResult asyncResult)
   at Yinrg.Net.Sockets.SocketClient.s91nev5CPITrFiQpWCt(Object , Object )
   at Yinrg.Net.Sockets.SocketClient.GsnOTWuuJ(IAsyncResult  )
